### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by cooljeanius**
I was wondering why I couldn't merge #312, and it seems that it's because I removed myself as a codeowner for the release workflow in #293; this adds me back.

### CODE REVIEW

**Review by rougetimelord — APPROVED** [review]
LGTM

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -15,6 +18,7 @@ src/* @kheina @jaredcat @cooljeanius @rougetimelord
 .prettierrc @kheina @jaredcat
 CHANGELOG.md @kheina @rougetimelord
 LICENSE @kheina
+privacy.md @kheina @rougetimelord
```
Comment (cooljeanius) [inline .github/CODEOWNERS:21]: oh actually I guess you can feel free to add me to this one, too, since I copyedited it...

Comment (rougetimelord) [reply .github/CODEOWNERS]: Oops yea, I got you

**Comment by cooljeanius** [comment]
Darn it, looks like @kheina still needs to approve... @rougetimelord would it be ok if I added you to the line for the CODEOWNERS file itself?

**Review by kheina — APPROVED** [review]

**Pull request merged**